.circleci/config.yml

```
@@ -51,6 +51,9 @@ jobs:
             - v1-0-jest-cache-{{ checksum "package.json" }}
             # fallback to using the latest cache if no exact match is found
             - v1-0-jest-cache-
+      - run:
+          name: Add .env file (react-native-dotenv)
+          command: yarn create-dotenv
```

```suggestion
          command: cp .env-sample .env
```
This will be create the `.env` from `.env-sample` wich is better then run it with production `.env`

---

Done! thanks